### PR TITLE
libmxnet v2.0.0.beta0.rc1 (new formula)

### PR DESCRIPTION
**Pull request by ngam**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Note: 
- unsure how best to test; currently builds fine locally, but will need help ensuring misc deps get packaged (or not packaged) correctly. Thanks! 
- currently this is only tested on darwin (copying darwin-specific cmake config); if things go well, we could relax the build

### CODE REVIEW

File: Formula/libmxnet.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class Libmxnet < Formula
+  desc "Deep Learning with Dynamic, Mutation-aware Dataflow Dep Scheduler"
+  homepage "https://mxnet.apache.org"
+  url "https://github.com/apache/incubator-mxnet.git",
+      tag:      "v2.0.0.beta0.rc1",
```
Comment (SMillerDev) [inline Formula/libmxnet.rb:5]: https://docs.brew.sh/Acceptable-Formulae#stable-versions

**Review by SMillerDev — COMMENTED** [review]
We don't ship pre-release software.

File: Formula/libmxnet.rb (hunk 1 above)
Comment (ngam) [reply Formula/libmxnet.rb:5]: Can one do "head-only" then or no?

The reasoning behind the beta here is twofold: 
- 2.x release seems to be nearing upstream
- 1.x release has outdated 3rd party deps (mainly onednn) that will require additional unnecessary work to get it work (and there is no reason to bother with it if 2.x is coming soon)

Also, any other feedback if I were to resubmit this again?

Comment (SMillerDev) [reply Formula/libmxnet.rb:5]: Head-only is still not a stable release so unfortunately no.

Comment (ngam) [reply Formula/libmxnet.rb:5]: Thanks! I will consider submitting this again in the future :)

**Pull request closed without merging**